shell/src/shared/molecules/Select/List.svelte

```diff
@@ -174,15 +174,8 @@ function handleKeyDown(e) {
 }
 
 function scrollToActiveItem(className) {
```

Comment:
Please rename the function to something like "scrollInputIntoView()" and remove the unused parameter.